data/dock.metainfo.xml.in

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 elementary, Inc. <contact@elementary.io> -->
+<component>
+  <id>io.elementary.dock</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <compulsory_for_desktop>Pantheon</compulsory_for_desktop>
+
```

Comment:
Maybe we may want to add `<translation type="gettext">io.elementary.dock</translation>` somewhere in the metainfo file to prevent the `May Not Be Translated` message from showing in AppCenter?

![Screenshot from 2024-05-11 08-36-53](https://github.com/elementary/dock/assets/26003928/14922fdf-de38-4870-b47b-661038cbacec)